packages/server/src/project/project.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Post, Get, Body, Param, Put } from '@nestjs/common';
+import { Prisma, Project } from '@prisma/client';
+import { ProjectIdentifier, ConfigurableProjectSettings } from './dto/project.dto';
+import { ProjectService } from './project.service';
+
+@Controller('/project')
+export class ProjectController {
+  constructor(private projectService: ProjectService) {}
+
+  @Post('/create')
+  createProject(@Body() newProject: Prisma.ProjectCreateInput): Promise<Project> {
+    return this.projectService.createProject(newProject);
+  }
+
+  @Get('/list')
```

Comment:
`/projects` should just list projects 
```suggestion
  @Get()
```